tests/test_bgp_af.rb

```diff
@@ -295,16 +316,25 @@ def test_collection_empty
   ##
   def test_collection_not_empty
     config('feature bgp') if platform == :nexus
-    config('router bgp 55',
-           'address-family ipv4 unicast',
-           'vrf red',
-           'address-family ipv4 unicast',
-           'vrf blue',
-           'address-family ipv6 multicast',
-           'vrf orange',
-           'address-family ipv4 multicast',
-           'vrf black',
-           'address-family ipv6 unicast')
+
+    bgp_afs = []
+    %w(default red blue orange black).each do |vrf|
+      [%w(ipv4 unicast), %w(ipv6 unicast), %w(ipv4 multicast), %w(ipv6 multicast)].each do |af|
+        config_ios_xr_dependencies(55, vrf)
+        bgp_afs.push(RouterBgpAF.new(55, vrf, af))
+      end
+    end
+
+    # config('router bgp 55',
```

Comment:
@chburket - You suggested I add this function; is this what you were thinking about, or did I miss the mark?